tests/integration-tests/tests/efa/test_efa/test_efa/pcluster.config.yaml

```diff
@@ -30,7 +30,7 @@ Scheduling:
             {% if instance == "p4d.24xlarge" %}GdrSupport: true{% endif %}
 {% if instance == "p4d.24xlarge" %}
           CapacityReservationTarget:
-            CapacityReservationId: cr-0fa65fcdbd597f551
+            CapacityReservationResourceGroupArn: arn:aws:resource-groups:us-east-1:447714826191:group/EC2CRGroup
```

Comment:
we should have a test for both

Reply:
A dedicated integration test is planned in a separate PR